packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -302,6 +308,13 @@ export class DataTable {
     }
   }
 
+  @Watch('colResize')
+  handleColResize() {
+    setTimeout(() => {
+      this.colResize && this.setResizerHeight();
```

Comment:
can we rename it to setColResizerHeight to make it more descriptive?

Reply:
Done